lib/temporal/configuration.rb

```diff
@@ -50,5 +51,13 @@ def task_list=(name)
     def timeouts=(new_timeouts)
       @timeouts = DEFAULT_TIMEOUTS.merge(new_timeouts)
     end
+
+    def grpc_ssl_config
+      if @channel_creds.nil?
+        :this_channel_is_insecure
```

Comment:
also would be great to have a spec for this and also put the value in a constant